src/app/containers/manuscript/affiliations-list/__tests__/__snapshots__/affiliations-list.spec.tsx.snap

```diff
@@ -23,7 +23,7 @@ exports[`Authors details info renders affiliations list 1`] = `
       <div
         className="makeStyles-affiliationInfo-3"
       >
-        Production Department, eLife Sciences, Cambridge, United Kingdom
+        [object Object], Cambridge, United Kingdom
```

Comment:
This looks like a regression, test needs updating to cope with the merge of dept/institution.